src/ucs/arch/cpu.h

```diff
@@ -18,6 +18,21 @@ typedef enum ucs_cpu_model {
     UCS_CPU_MODEL_LAST
 } ucs_cpu_model_t;
 
+/* CPU flags */
+typedef enum ucs_cpu_flag {
+    UCS_CPU_FLAG_UNKNOWN    = (-1),
```

Comment:
enum is signed interger, so I would like to have some value as invalid. Of course I do not want to compare bits with UCS_CPU_FLAG_UNKNOWN.